releasenotes/notes/u_U-535517c8ab772e16.yaml

```
@@ -0,0 +1,4 @@
+---
+upgrade:
+  - |
+    The gate :class:`qiskit.circuit.library.standard_gates.u.UGate` was named from `u` to `U`.
```

When I read this I thought the gate was changed from `UGate` to `U`. Maybe this would be a little clearer?
```suggestion
    The name of the gate :class:`qiskit.circuit.library.standard_gates.u.UGate` was changed from `"u"` to `"U"`.
```